src/Sylius/Bundle/CurrencyBundle/Resources/config/services.xml

```diff
@@ -31,10 +31,10 @@
 
         <service id="sylius.templating.helper.currency" class="Sylius\Bundle\CurrencyBundle\Templating\Helper\CurrencyHelper" lazy="true">
             <tag name="templating.helper" alias="sylius_currency" />
+            <deprecated package="sylius/sylius" version="1.14">The "%service_id%" is deprecated since Sylius 1.14 and will be removed in 2.0.</deprecated>
```

Comment:
```suggestion
            <deprecated package="sylius/currency-bundle" version="1.14">The "%service_id%" is deprecated since Sylius 1.14 and will be removed in 2.0.</deprecated>
```
Shouldn't it here and also in other places be also targetted against the same bundles?